aioodbc/pool.py

```diff
@@ -21,6 +23,7 @@ class Pool:
 
     def __init__(
         self,
+        dsn: str,
```

Comment:
Why this extra parameter was added? This is breaking change in API.

Reply:
`dsn` is implicitly passed to `connect()` in `_fill_free_pool()` as part of `_conn_kwargs` in two places:
- https://github.com/aio-libs/aioodbc/blame/5f8961d198de7e553e95f8e050ad36b25c098b39/aioodbc/pool.py#L163
- https://github.com/aio-libs/aioodbc/blame/5f8961d198de7e553e95f8e050ad36b25c098b39/aioodbc/pool.py#L175

`_conn_kwargs` contains the Pool `__init__` method's kwargs:

https://github.com/aio-libs/aioodbc/blob/3f646e308db20c44a84e698bb5c235d91a0a9677/aioodbc/pool.py#L47

If a `dsn` kwarg is not passed during the creation of a Pool object, a TypeError will be raised in `acquire()`:

> TypeError: connect() missing 1 required keyword-only argument: 'dsn'

In other words, `dsn` is already required; it's just missing in the definition.